price-feeder/CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - [#1032](https://github.com/umee-network/umee/pull/1032) Update the accepted tvwap period from 3 minutes to 5 minutes.
 - [#978](https://github.com/umee-network/umee/pull/978) Cleanup the oracle package by moving deviation & conversion logic.
 - [#1050](https://github.com/umee-network/umee/pull/1050) Cache x/oracle params to decrease the number of queries to nodes.
+- [#1069](https://github.com/umee-network/umee/pull/1069) Cache chain height to subscribe to node event EventNewBlockHeader.
```

Comment:
Is this really a _cache_ in the normal sense anymore? I tend to associate a cache with latency - wdyt?
Let's at least make sure that the code & comments are clear that the block height is always up-to-date.

Reply:
updated